### PR TITLE
remove some site configuration, non used checkstyle profile and turn skip xml into one line property

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: Olivier Lamy <oliver.lamy@gmail.com>

### CODE REVIEW

File: jetty-jspc-maven-plugin/pom.xml [hunk 1]
```diff
@@ -11,6 +11,8 @@
   <name>Jetty :: Jetty JSPC Maven Plugin</name>
   <properties>
     <bundle-symbolic-name>${project.groupId}.jspc.plugin</bundle-symbolic-name>
+    <jacoco.skip>true</jacoco.skip>
+    <maven.test.skip.exec>true</maven.test.skip.exec>
```
Comment (joakime) [inline jetty-jspc-maven-plugin/pom.xml]: Why `maven.test.skip.exec` here?

**Review by joakime — CHANGES_REQUESTED** [review]
One odd new property that doesn't appear before this.

File: jetty-jspc-maven-plugin/pom.xml (hunk 1 above)
Comment (olamy) [reply jetty-jspc-maven-plugin/pom.xml]: ah yeah good catch. I mixed some stuff when doing the same thing for 2 branches which are different sorry for that

Comment (olamy) [reply jetty-jspc-maven-plugin/pom.xml:15]: oh but wait the surefire plugin configuration.
see next commit :)

**Review by joakime — APPROVED** [review]
LGTM

**Pull request merged**